ui/src/router/components/form/validation/schema.js

```diff
@@ -221,7 +221,7 @@ const resourceRequestSchema = (maxAllowedReplica) =>
         maxAllowedReplica,
         "Max Replicas value has exceeded allowed number of replicas: ${max}"
       )
-      .when("min_replica", (minReplica, schema) =>
+      .when("min_replica", ([minReplica], schema) =>
```

Comment:
what is the impact if we don't make this change?

Reply:
Ah, the direct impact would be `minReplica` being a list with 1 integer (the actual `min_replica` value, e.g. `[0]`), which will cause the comparison operator `minReplica === 0` to fail, thus allowing `max_replica` to be set as 0. In other words, our users will get to set min_replica=max_replica=0 which isn't a big problem but probably something we can block upfront.